### PR TITLE
Using TK on Supermatter now burns your head off.

### LINKED ISSUE #25870: you can touch sm with tk and it'll dust you
kek

### PR DESCRIPTION
**Pull request by RemieRichards**
REALLY Fixes #25870.
dumb coiax ( !!FUN!! alternative to #25871 )

:cl: RemieRichards
fix: Using TK on the supermatter will burn your head off violently, don't do this.
/:cl:

### CODE REVIEW

**Comment by Razharas** [comment]
I actually like this idead

**Comment by RemieRichards** [comment]
so do I but I just know people will miss the joke and think it's a real typo (which it originally was but oh well)

**Comment by Razharas** [comment]
Arent we in freeze though?

**Comment by RemieRichards** [comment]
It's a fix. (technically it's elevating a bug to a feature, which is "fixing" the bug)

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -419,6 +419,13 @@
 			"<span class='italics'>You hear a loud crack as you are washed with a wave of heat.</span>")
 			Consume(B)
 
+/obj/machinery/power/supermatter_shard/attack_tk(mob/user)
+	if(iscarbon(user))
```
Comment (Jalleo) [inline code/modules/power/supermatter/supermatter.dm]: Not supporting silicon TK

**Review by Jalleo — CHANGES_REQUESTED** [review]
I suggest making it possible for TK silicons to get burned too

**Comment by lzimann** [comment]
silicons can't have TK

**Comment by RemieRichards** [comment]
silicons don't have DNA, so they can't have mutations, so they can't have TK.

**Pull request merged**